Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist

```diff
@@ -441,7 +441,59 @@ Note that Google does not support the offline_access scope so instead use the pr
 		</dict>
 		<dict>
 			<key>pfm_default</key>
-			<integer>0</integer>
+			<false/>
+			<key>pfm_description</key>
+			<string>Favor using XCreds' local login screen over the cloud login UI.</string>
+			<key>pfm_documentation_url</key>
+			<string>https://twocanoes.com/knowledge-base/xcreds-admin-guide/#preferences</string>
+			<key>pfm_name</key>
+			<string>shouldPreferLocalLoginInsteadOfCloudLogin</string>
+			<key>pfm_title</key>
+			<string>Prefer Local Login over Cloud Login</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_description</key>
+			<string>When verifying password in the menu app, use ROPG.</string>
+			<key>pfm_documentation_url</key>
+			<string>https://twocanoes.com/knowledge-base/xcreds-admin-guide/#preferences</string>
+			<key>pfm_name</key>
+			<string>shouldVerifyPasswordWithRopg</string>
+			<key>pfm_title</key>
+			<string>Use ROPG when testing password</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>ROPG Client ID for use when checking password</string>
```

Comment:
Looking at the manifest, most descriptions end with a period. Let's add one here too?